chickadee/processes/wps_CI.py

```diff
@@ -0,0 +1,144 @@
+from pywps import Process, ComplexInput, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+from pywps.app.exceptions import ProcessError
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, get_ClimDown
+
+
+class CI(Process):
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+        inputs = [
+            log_level,
+            ComplexInput(
+                "gcm_file",
+                "GCM NetCDF file",
+                abstract="Filename of GCM simulations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF],
+            ),
+            ComplexInput(
+                "obs_file",
+                "Observations NetCDF file",
+                abstract="Filename of high-res gridded historical observations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF],
+            ),
+            LiteralInput(
+                "varname",
+                "Variable to Downscale",
+                abstract="Name of the NetCDF variable to downscale (e.g. 'tasmax')",
```

Comment:
Do we have any limitations on variables here? Should there be a set of options or some sort of check the ensure they are passing a valid var? I imagine this may be handled somewhere down the line that I have not seen yet.

Reply:
ClimDown handles the validity of the error:
```
> ClimDown::ci.netcdf.wrapper('./tiny_gcm.nc', './tiny_obs.nc', "ci.nc", "tasmin")
[1] "vobjtovarid4: error #F: I could not find the requsted var (or dimvar) in the file!"
[1] "var (or dimvar) name: tasmin"
[1] "file name: ./tiny_gcm.nc"
Error in vobjtovarid4(nc, varid, verbose = verbose, allowdimvar = TRUE) : 
  Variable not found
```